src/System.Runtime.InteropServices/ref/System.Runtime.InteropServices.cs

```diff
@@ -403,6 +403,10 @@ public static partial class Marshal
         public static string PtrToStringUni(System.IntPtr ptr) { return default(string); }
         [System.Security.SecurityCriticalAttribute]
         public static string PtrToStringUni(System.IntPtr ptr, int len) { return default(string); }
+        [System.Security.SecurityCriticalAttribute]
+        public static string PtrToStringUTF8(System.IntPtr ptr) { return default(string); }
+        [System.Security.SecurityCriticalAttribute]
+        public static string PtrToStringUTF8(System.IntPtr ptr, int byteLen) { return default(string); }        
```

Comment:
Yes , It is deliberate since what we expect is really length in bytes . Naming it len is confusing since it not quite clear if it's length in char or bytes